packages/miniflare/test/plugins/d1/suite.ts

```diff
@@ -392,10 +403,29 @@ test("D1PreparedStatement: raw", async (t) => {
 		.bind("yellow")
 		.first("id");
 	t.is(id, 4);
+
+	// Check whether workerd raw test case passes here too
+	// Note that this test did not pass with the old binding
+	if (!t.context.bindings["__D1_BETA__DB"]) {
```

Comment:
This test suite is dynamic (yay) and gets run by both versions of the D1 binding.

I had to add this check to ensure that it's fixed only for new bindings (since old versions of wrangler have the old binding)

Reply:
Fascinating!